### ISSUE TITLE
transformDexArchiveWithExternalLibsDexMergerForDebug while using expo and tipsi-stripe libs in same app

### ISSUE THREAD

**Issue by paynd** [expo/expo#967]
User got:
`Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'.`
When tried to use expo and tipsi-stripe at the same time.
As I can see https://github.com/expo/expo/tree/master/android/stripeview/stripeview/src/main/java/com/gettipsi/stripe 
For example `com.gettipsi.stripe.StripeModule` has equal name in both libraries.
Would you change namespace of copied code?

**Comment by ide** [comment]
We're working on a more robust way to namespace libraries but it may take a few months or more. So if you can find a workaround that's probably the best way to get unblocked.

**Comment by waqqas** [comment]
@ide can you recommend any way for the workaround?

**Comment by paynd** [comment]
@waqqas You can fork expo, move stripe packages to another namespace and use it until problem is resolved.

**Comment by ide** [comment]
Yeah, I don't think there is a clean workaround / not sure of one off the top of my head.

**Comment by waqqas** [comment]
@ide any update on this?

**Comment by ide** [comment]
@waqqas I don’t have any updates unfortunately. The Expo team isn’t looking into this and figuring out a workaround is probably the best path forward.

**Comment by rafaelwkerr** [comment]
I had the same problem and fixed with this solution:

http://blog.csdn.net/fanszheqq/article/details/78357323

**Comment by affanshahab** [comment]
@paynd can you please further elaborate on how to fork expo and use it in app to get it work. I have spent days and couldn't find a working solution. I also have sent you an email on similar topic. Please check your junk too. Thanks

**Comment by waqqas** [comment]
@ide is this bug fixed in the expo version 25.0.0?

**Comment by ide** [comment]
No, this was not part of SDK 25.

**Comment by sjchmiela** [comment]
I think this issue does not appear in recent versions of SDK. Thanks for submitting it, I'll let myself close it for now, but feel free to open a new issue if you encounter this issue again! 🙂
